woocommerce/wc-admin
# Fix CurrencyFactory constructor to use proper function

Fixes #7260 

Change CurrencyFactory constructor to use a proper function instead of an arrow function to guarantee the build won't be producing an arrow function for a constructor.

### Detailed test instructions:

1.  Run `npm run clean && npm start` to make sure a clean build.
1.  Open up the built `dist/currency/index.js` file.
2. Search for the `CurrencyFactory` constructor and make sure it's not an arrow function.
